events/blocks/event-subscription-form/event-subscription-form.js

```diff
@@ -102,6 +103,15 @@ function decorateButton(bp) {
 }
 
 function addElementToForm(form, inputP, labelP) {
+  const profile = BlockMediator.get('imsProfile');
+  if (profile === undefined) {
+    window.lana?.log('No profile found');
+    BlockMediator.subscribe('imsProfile', (data) => {
+      if (data && data.email) {
+        form.querySelector('.subscription-input').value = data.email;
```

Comment:
Nit: best not to assume a querySelector will always find what it's looking for. I would wrap this in an if before trying to access the .value.